### ISSUE TITLE
<select> input tag not working in Django

### ISSUE THREAD

**Issue by Erwol** [Dogfalo/materialize#4904]
### Description
`<select>` tags are not working in Django because they need the  class="browser-default" to appear in the browser.

### Repro Steps

1. Create a Form with multiple choices in Django
2. Try to render it; you won't be able to see it

The solution is to overwrite the forms widget, but I think this should be fixed from materialize css project.

```python
class JustAnotherForm(forms.ModelForm):
    multiplechoicefield= forms.CharField(
        max_length=124,
        widget=forms.Select(
            choices=ARTWORK_TYPES,
            attrs={'class': 'browser-default'}),
        
    )
```

**Comment by DanielRuf** [comment]
Can you provide a codepen which reproduces it? Did you already inspect the element in the developer tools of the browser to find the problematic code?

**Comment by Dogfalo** [comment]
The select is meant to be initialized which you are not doing. In any case, we likely will make some changes to default elements in the upcoming releases

**Comment by DanielRuf** [comment]
Right, should be initalized with `material_select()` (I thought he did so in his project or component).

http://materializecss.com/forms.html

**Comment by Erwol** [comment]
Hey @DanielRuf, I already found the problem when I post the issue. Sorry if it's not well explained. The thing is that Django renders `select` tags without any classes attached to the element, and as materialize css needs the _browser-default_ set class in select tags, Django will render the template but you won't be able to see the select. The Django-side fix is to initialize the component as I posted up here. Maybe an easy fix'd be to allow native select tags. Thank you @Dogfalo.
